python/paddle/autograd/functional.py

```diff
@@ -904,3 +904,73 @@ def func(x, y):
         vhp = grad_fn(jac, xs, v)
         outputs, vhp = return_fn(outputs), return_fn(vhp)
     return outputs, vhp
+
+
+from paddle.static import gradients
+class Jacobian(object):
```

Comment:
Maybe the "from xx import xx" statement should be declared in the begging of this file?

Reply:
Done.